### DISCUSSION TITLE
How to stack multiple relation manager tables instead of tabs?

### DISCUSSION THREAD

**Discussion by joshembling** [filamentphp/filament#6718] [General]
I have 3 relation managers, which are viewed in tabs by default instead of stacked tables. 

How can I reverse this so that I have multiple tables viewable on scroll. 

The opposite of this? Move tabs to stacked relation manager tables https://filamentphp.com/docs/2.x/admin/resources/relation-managers#moving-the-resource-form-to-tabs

**Comment by danharrin** [comment]
You can wrap all relation managers in 1 RelationGroup

**Reply by mrleblanc101** [reply]
Is there a way to make this the default ?
Just like there is a way to make them editable by default `->readOnlyRelationManagersOnResourceViewPagesByDefault(false)`.
Would be nice to have a `->stackRelationshipManagerByDefault(true)`

**Reply by mrleblanc101** [reply]
I've created a feature request here: https://github.com/filamentphp/filament/discussions/18545
